phalcon/mvc/model/criteriainterface.zep

```diff
@@ -192,6 +193,6 @@ interface CriteriaInterface
 	/**
 	 * Executes a find using the parameters built with the criteria
 	 */
-	public function execute() -> <ResultsetInterface>;
+	public function execute(<TransactionInterface> transaction = null) -> <ResultsetInterface>;
```

Comment:
Idk who and why, this just breaks BC, and since from phalcon 3 we are following this the interface change can't be done for 3.x.0 version. Only for new major version. Not sure about argument in class with default value, maybe this is acceptable @sergeyklay